### PR TITLE
Fix #2326 - Move Twitter and Facebook feed under one fragment.

### LINKED ISSUE #2326: Move both Twitter and Facebook feed under one fragment
**Actual Behaviour**

Currently, there are two separate items in the drawer for twitter and facebook.

**Expected Behaviour**

I propose to move both the fragments under a common "Live Feed" fragment using a `View Pager`.

**Would you like to work on the issue?**

Yes

@iamareebjamal @srv-twry Please see if this is a good change and is it required atm?

### PR DESCRIPTION
**Pull request by mayank8318**
Fixes #2326 
Changes: Made a new fragment with a view pager and tab layout which shows the two feed fragments.
Also refactored feedFragment to FacebookFeedFragment.

Screenshots of the change: 
image

### CODE REVIEW

File: android/app/src/main/java/org/fossasia/openevent/core/feed/FeedFragment.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        feedTabLayout.setupWithViewPager(viewPager);
+
+        return view;
+    }
+
+    private void setUpViewPager(ViewPager viewPager) {
+        FeedViewPagerAdapter feedViewPagerAdapter = new FeedViewPagerAdapter(getChildFragmentManager());
+        feedViewPagerAdapter.addFragment(FacebookFeedFragment.getInstance(openCommentsDialogListener, onImageZoomListener), "FACEBOOK");
+        feedViewPagerAdapter.addFragment(TwitterFeedFragment.getInstance(onImageZoomListener), "TWITTER");
+        feedViewPagerAdapter.notifyDataSetChanged();
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/core/feed/FeedFragment.java]: Use strings.xml and don't capitalize by default, it is handled automatically by Andriod Tab Layout

**Review by iamareebjamal — CHANGES_REQUESTED** [review]

**Comment by mayank8318** [comment]
@iamareebjamal Please review :)

File: android/app/src/main/java/org/fossasia/openevent/core/feed/FeedFragment.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+import butterknife.BindView;
+
+public class FeedFragment extends BaseFragment {
+
+    @BindView(R.id.viewpager)
+    protected ViewPager viewPager;
+    @BindView(R.id.tabLayout)
+    protected TabLayout feedTabLayout;
+
+    private FacebookFeedAdapter.OpenCommentsDialogListener openCommentsDialogListener;
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/core/feed/FeedFragment.java]: If it is general, why is it inside FacebookFeedAdapter

**Review by iamareebjamal — CHANGES_REQUESTED** [review]

File: android/app/src/main/java/org/fossasia/openevent/core/feed/FeedFragment.java (hunk 2 above)
Comment (mayank8318) [reply android/app/src/main/java/org/fossasia/openevent/core/feed/FeedFragment.java]: @iamareebjamal 
Presently only Facebook comments are shown. Twitter feed doesn't support comments as of now.
Should I move it to a general location and make an issue for adding comments to twitter feed?

Comment (iamareebjamal) [reply android/app/src/main/java/org/fossasia/openevent/core/feed/FeedFragment.java]: If it is used in FeedFragment, it should be in general package, let's not show twitter comments but just move it to general package

**Comment by mayank8318** [comment]
Changes: Move openCommentsDialogListener to an independent interface.
@iamareebjamal Please review :)

**Review by iamareebjamal — APPROVED** [review]
Good job

**Pull request merged**